lib/dish.rb

```diff
@@ -0,0 +1,7 @@
+class Dish
+  attr_reader :price, :name
+  def initialize(name, price)
```

Comment:
Good use of multiple classes/files

Reply:
Although the code is easy to follow and very clear, You could add a README to explain your approach to the challenge and how to use your program.